Methods/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Methods
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine(CalculateFormula(2, 3) + CalculateFormula(5, 7));

```

Comment:
Округлять полученное окончательное значение выражения следует только в момент вывода, а не при промежуточных вычислениях. Ив данной ситуации для облегчения чтения и пони мания кода лучше ввести локальную переменную
```suggestion
            double result = CalculateFormula(2, 3) + CalculateFormula(5, 7);
            Console.WriteLine(Math.Round(result, 3));
```